spec/support/admin_view_claim_feature_shared_examples.rb

```diff
@@ -34,7 +34,12 @@
   }
 
   let!(:similar_claim) {
-    eligibility = create(:"#{policy.to_s.underscore}_eligibility", :eligible, teacher_reference_number: multiple_claim.eligibility.teacher_reference_number)
+    duplicate_attribute = if policy == Policies::InternationalRelocationPayments
+      {passport_number: multiple_claim.eligibility.passport_number}
```

Comment:
👍 
I was wondering if we could use the national insurance number as it's required for all journeys and it should be unique. Obv passport number is fine too